### ISSUE TITLE
EventSub documentation for Goals differ and is harder to read

### ISSUE THREAD

**Issue by Emilgardis** [twitchdev/issues#502]
The docs for EventSub Goals differ a lot from the other events.

Starting at [Subscription Types](https://dev.twitch.tv/docs/eventsub/eventsub-subscription-types)

Clicking these
![image](https://user-images.githubusercontent.com/1502855/133890557-7685f24c-efa0-4503-ba97-c97c3f61d4bb.png)
brings you to [`Creator goals begin event`](https://dev.twitch.tv/docs/eventsub/eventsub-subscription-types#begin-goals-event), not `Goals Begin` etc. The link should probably be `Channel Goal Begin` or `Creator...`
![image](https://user-images.githubusercontent.com/1502855/133890570-19fdf2e8-f57c-42d1-bda1-23fa9d05c1ed.png)

The anchor for these are called `{begin,progress,end}-goals-event`, which is not consistent with the other events/notifications.

Furthermore, the documentation for these entries is hard to parse compared to the other entries

![image](https://user-images.githubusercontent.com/1502855/133890629-3a41cd20-abb7-4727-aa3e-c7667362e289.png)

Something like the other entries would be better
![image](https://user-images.githubusercontent.com/1502855/133890717-c625f0bb-1907-40ec-9725-16fb852895ad.png)

**Additional context or questions**

**Comment by jbulava** [comment]
Creator Goals subscription types have been updated to use the format of other subscription types for consistency. These updates will be included in today's deployment.
